config/names/dreams.txt

```diff
@@ -0,0 +1,106 @@
+an ID carda bottle
+a familiar face
+a crewmember
+a toolbox
+a security officer
+the captain
+voices from all around
+deep space
+a doctor
+the engine
+a traitor
+an ally
+darkness
+light
+a scientist
+a monkey
+a catastrophe
+a loved one
+a gunwarmth
+freezing
+the sun
+a hat
+the Cyberiad
+a ruined station
+a planet
+plasma
+air
+the medical bay
+the bridge
+blinking lights
+a blue light
+an abandoned laboratory
+Nanotrasen
+The Syndicate
+blood
+healing
+power
+respect
+riches
+spacea crash
+happiness
+pride
+a fall
+water
+flames
+ice
+melons
+flying
+the eggs
+money
+the head of personnel
+the head of security
+a chief engineer
+a research director
+a chief medical officer
+the detective
+the warden
+a member of the internal affairs
+a station engineer
+the janitor
+atmospheric technician
+the quartermaster
+a cargo technician
+the botanist
+a shaft miner
+the psychologist
+the chemist
+the geneticist
+the virologist
+the roboticist
+the chef
+the bartender
+the chaplain
+the librarian
+the brig physician
+the pod pilot
+the barber
+the mechanic
+the magistrate
+the nanotrasen representative 
+the blueshield 
+a mouse
+an ert member
+a beach
+the holodeck
+a smokey room
+the cold
+an operating table
+the bar
+the rain
+a skrell
+a unathi
+a tajaran 
+a vulpkanin 
```

Comment:
I think this should be under nightmares.txt 😛

Reply:
:running_man: :point_left: :fox_face: :hocho:

Reply:
you knock that off..